lib/msal-common/src/server/ServerAuthorizationCodeResponse.ts

```diff
@@ -8,13 +8,23 @@
  * - code: authorization code from server
  * - client_info: client info object
  * - state: OAuth2 request state
+ * - cloud_instance_name: cloud authority name
+ * - cloud_instance_host_name: host uri of cloud authority
+ * - cloud_graph_host_name: host uri of cloud graph instance
+ * - msgraph_host: host uri of MSGraph Instance
  * - error: error sent back in hash
  * - error: description
  */
 export type ServerAuthorizationCodeResponse = {
+    // Success case
     code?: string;
     client_info?: string;
     state?: string;
+    cloud_instance_name?: string;
+    cloud_instance_host_name?: string;
+    cloud_graph_host_name?: string;
+    msgraph_host?: string;
```

Comment:
Why does graph need to be included in this?

Reply:
These are the parameters sent back when instance_aware=true, I believe it is because the graph host is different for different clouds. We may need to add this to the response in the future, for now I am just adding it as part of the params  sent back in the hash.

Reply:
I feel like we shouldn't be coding against specific resources. I'm not totally clear on what "instance aware" means in this context but are you saying that the developer would not know which graph host they need to hit without this in the response? Shouldn't the developer be responsible for knowing their resource endpoints?